src/current/releases/molt.md

```diff
@@ -40,6 +40,20 @@ For more information, refer to [Configuration]({% link molt/live-migration-servi
 </section>
 
 <section class="filter-content" markdown="1" data-scope="fetch_verify">
+## July 31, 2024
+MOLT Fetch/Verify 1.1.3 is [available](#installation).
+
+- `'infinity'::timestamp` values can now be moved with Fetch.
+- Fixed an issue where connections were not being closed immediately after sharding was completed. This could lead to errors if the [maximum number of connections]({% link molt/molt-fetch.md %}#best-practices) was set to a low value.
+- Fetch users can now exclude specific tables from migration using the [`--table-exclusion-filter` flag]({% link molt/molt-fetch.md %}#global-flags).
+
```

Comment:
@ryanluu12345 Did your computed columns support not have a Release Note?

Reply:
Hmm I guess not, but I also made a separate ticket for computed columns support. @taroface can you confirm you have that computed column ticket handy?